src/clj/game/cards/events.clj

```diff
@@ -809,6 +820,12 @@
       :effect (req (show-wait-prompt state :corp "Runner to rearrange the top cards of their stack")
                    (let [from (take 6 (:deck runner))]
                      (continue-ability state side (entrance-trash from) card nil)))})
+   
+   "Mars for Martians"
+   {:msg (msg "draw " (count (filter #(and (has-subtype? % "Clan")
+                                           (is-type? % "Resource")) (all-installed state :runner))) " card(s) and gain " (:tag runner) " [Credits]")
+    :effect (effect (draw (count (filter #(and (has-subtype? % "Clan")
+                                               (is-type? % "Resource")) (all-installed state :runner)))) (gain :credit (:tag runner)))}
```

Comment:
Should look like this:  https://pastebin.com/qMmK490T

And either just say "cards" in the message or look at a card like Daily Business Show for a `when` statement in the message to determine if plural is necessary.